android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AccountRecord.java

```diff
@@ -6,10 +6,15 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import java.util.Arrays;
+
 /* package */ class AccountRecord {
     @IntRange(from = 1)
     final int id;
 
+    @IntRange(from = 1)
+    final int parentId;
```

Comment:
To clarify, the `parentId` of an account will always be the id of the "authRecord" that the account is associated with, correct?

Curios, why its called `parentId` instead of `authRecordId`? Is there another possible value it can be?

Reply:
to align with the schema. Parent id is used to indicate that this is a subtable